bull-bean-transformer/src/test/java/com/hotels/beans/transformer/ImmutableObjectTransformationTest.java

```diff
@@ -500,6 +513,33 @@ public void testTransformRaiseAnExceptionIfTheTransformerFunctionIsNotValid() {
 
     }
 
+    /**
```

Comment:
As this test refers to a Mutable object, please move it in the class: `MutableObjectTransformationTest`

Reply:
This is done.